src/command/init/options/project_use_case.rs

```diff
@@ -48,12 +48,17 @@ pub enum ProjectUseCase {
     GraphQLTemplate,
 }
 
+#[cfg(feature = "init")]
+const USE_CASE_DESCRIPTION: &str = "Select the type of project to start";
```

Comment:
We moved away from calling it 'project' and are keeping it as 'graph' for now.  I think we want to stick with the other copy minus the '- coming soon' tag
```suggestion
const USE_CASE_DESCRIPTION: &str = "Start a graph with recommended libraries";
```